test/sbroker_fq_statem.erl

```diff
@@ -502,9 +502,6 @@ out(#state{timeout_time=Timeout, time=Time, robin=Robin}  = State) ->
 
 out_loop([], [], #state{timeout_time=infinity} = State) ->
     {empty, [], State#state{robin=[]}};
-out_loop([], Acc, #state{timeout_time=infinity} = State) ->
-    [FirstEmpty | Robin] = lists:reverse(Acc),
-    {empty, [], State#state{robin=Robin++[FirstEmpty]}};
```

Comment:
^ Why the removal? Did Dialyzer state this code was unreachable?

Reply:
Yes, that code is unreachable. And since tests are passing without it, I blindly trusted `dialyzer`.